scripts/tools/GlobalTimestamps.yml

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (C) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+#
+# See YaML.md for syntax definition
+#
+--- #--------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Level-Zero Tool Experimental Extension for Global Metrics Timestamps"
+version: "1.5"
+--- #--------------------------------------------------------------------------
+type: macro
+desc: "Global Metrics Timestamps Experimental Extension Name"
+version: "1.5"
+name: $T_GLOBAL_METRICS_TIMESTAMPS_EXP_NAME
+value: '"$XT_experimental_global_metrics_timestamps"'
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "Global Metrics Timestamps Experimental Extension Version(s)"
+version: "1.5"
+name: $x_global_metrics_timestamps_exp_version_t
+etors:
+    - name: "1_0"
+      value: "$X_MAKE_VERSION( 1, 0 )"
+      desc: "version 1.0"
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Returns global device timestamps synchronized with metrics timesstamps, optionally mapped to host."
```

Comment:
For better clarity I would invert the references. Something like 

`Returns metrics timestamps synchronized with global device timestamps`

leaving out host, since it is not in scope.

Reply:
@matcabral : if host are not in the scope, do you want to keep or remove the mappedToHost argument I added below?

```cpp
- type: "$x_bool_t"
      name: mappedToHost
      desc: "[in] Are timestamps and metrics mapped to host, or device."
```

Reply:
regarding host support, I saw below the boolean flag added.

Reply:
@matcabral : could you clarify, whether we want then to have host mapped support or not?

Reply:
yes, lets have it please. Confirmed also in comment below.